ruff_lsp/server.py

```diff
@@ -1192,13 +1539,9 @@ async def _run_check_on_document(
     )
 
 
-async def _run_format_on_document(document: workspace.TextDocument) -> RunResult | None:
+async def _run_format_on_document(document: Document) -> RunResult | None:
```

Comment:
Wow, so this works interchangeably with Python files and notebooks now?

Reply:
Yes, the `Document` object creates an abstraction to store all the information required in the format (specifically the source) it is required.